micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/CamundaBpmVersion.java

```diff
@@ -0,0 +1,21 @@
+package info.novatec.micronaut.camunda.bpm.feature;
+
+import org.camunda.bpm.engine.ProcessEngine;
+
+import java.util.Optional;
+
+public class CamundaBpmVersion {
+    private final String version;
+
+    public CamundaBpmVersion(){
+        this(ProcessEngine.class.getPackage());
+    }
+
+    CamundaBpmVersion(final Package pkg) {
```

Comment:
* please remove "final" here. It is not needed - even if it there in the original.
* please rename "pkg" to "package"

Reply:
Sorry, "package" is reserved. Just leave it as "pkg".